scripts/tokenomics-emulation/emulations/cere-app-to-user-emulation.ts

```diff
@@ -0,0 +1,29 @@
+import { IEmulation } from "./emulation.interface";
+import Accounts from "../accounts";
+import CereSmartContract from "../cere-smart-contract";
+
+class CereApptoUserEmulation implements IEmulation {
```

Comment:
As far as it is supporting app to user and vice versa it should be renamed. For instance, `CereIntegrationsEmulation`.

Reply:
done.